### DISCUSSION TITLE
Changing Twitter

### DISCUSSION THREAD

**Discussion by Concon87** [arcana-network/discussions#18] [Feature Request]
It is possible to change twitter account due to twitter suspension account?

**Comment by mmjee** [comment]
On Monday, January 22 2024 Anno Domini 10:21 AM India Standard Time, Concon87 wrote:
> It is possible to change twitter account due to twitter suspension account? 
> 
Hi,

Unfortunately, Twitter accounts cannot be changed once it is linked. Please contact Twitter and try to re-activate your account.
-- 
Regards,
